org.fundacionzt.undina.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.fundacionzt.undina</id>
+  <name>Undina</name>
+  <summary>Una aventura por el Océano</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <url type="homepage">https://fundacionzt.org/</url>
+  <description>
+    <p>
+      UNDINA – Una aventura por el Océano es una experiencia interactiva creada para inspirar a niñas, niños y adolescentes a proteger los océanos y su biodiversidad, entendiendo que cuidar el mar es clave para salvar nuestro planeta.
+    </p>
```

Comment:
primary language must be english, you can add translations with xml:lang https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines#metainfo-translations